contracts/eth/nearprover/hardhat.config.js

```diff
@@ -1,6 +1,28 @@
 require('@nomiclabs/hardhat-ethers');
 require('solidity-coverage');
 
+require('dotenv').config();
+
+const ETH_PRIVATE_KEY = process.env.ETH_PRIVATE_KEY;
+const INFURA_API_KEY = process.env.INFURA_API_KEY || '';
+
+task('upgrade-provers-bridge-address-to', 'Upgrades the provided prover to use the bridge at the provided address')
+    .addParam('prover', 'Prover address')
+    .addParam('newBridge', 'The address of the new bridge')
+    .setAction(async taskArgs => {
+        const { upgradeProversBridgeAddressTo } = require('./utils/upgrade_bridge_address.js');
+        await upgradeProversBridgeAddressTo(hre.ethers.provider, taskArgs.prover, taskArgs.newBridge);
+    });
```

Comment:
I'm used to use typescript for this, so the code looks a little bit weird. My editor is complaining that `task` and `hre` are not declared. In TS it is done the following way:

```suggestion
    .setAction(async (taskArgs, hre) => {
        const { upgradeProversBridgeAddressTo } = require('./utils/upgrade_bridge_address.js');
        await upgradeProversBridgeAddressTo(hre.ethers.provider, taskArgs.prover, taskArgs.newBridge);
    });
```

Reply:
@mfornet this is works because this script is being run with a hardhat command which provides all that needed env data. But yes, I agree it's better to have it suitable to run with the default commands too